tailwind.config.js

```diff
@@ -0,0 +1,35 @@
+const { colors } = require('./client/src/tokens/colors');
+const { systemUIFontStack } = require('./client/src/tokens/typography');
+const { fontSize } = require('./client/src/tokens/typography');
+const { breakpoints } = require('./client/src/tokens/breakpoints');
+
+module.exports = {
+  content: [
+    './wagtail/**!(static)/*.{html,js,jsx,ts,tsx}', // Avoid purging compiled static assets
+    './client/**/*.{html,js,ts,tsx}',
+    './docs/**/*.{html,js,ts,tsx}',
```

Comment:
With the way tailwind works now, classes in the files listed **will** be what shows up in the final css. It's sort of flipped from how it used to be where it would remove unused classes. Now it starts blank and only adds. 

Good catch though, I should probably add `md and rst` file types to this list?